Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN yarn install \
     && rm -rf /opt/app/.git
 
 # ---- Runtime image ----
+USER hmcts
```

Comment:
this doesn't look right the user is already hmcts and why would you put it here at the end of the build image? seems pointless

Reply:
Removed USER directive as hmcts is already used, thanks Tim.